test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -0,0 +1,140 @@
+import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
+import ChallengeFactorPageObject from '../framework/page-objects/ChallengeFactorPageObject';
```

Comment:
Let's extend `ChallengeFactorPageObject` into `ChallengEmailPageObject` (or ChallengeAuthenticatorEmailPageObject) which holds methods that are specify for email page, e.g. ResendEmail.
`ChallengeFactorPageObject` probably shall be clean up but you can leave annotation like `@deprecated` to those functions that are moved to sub class.